mne/viz/_brain/_brain.py

```diff
@@ -437,13 +438,16 @@ def time_label(x):
                                        vmin=dt_min,
                                        vmax=dt_max,
                                        scalars=act_data)
+            self._data[hemi + '_mesh'] = m
+            self._data[hemi + '_pd'] = p
```

Comment:
I don't understand the name `pd` even after looking at the code, what is it? It would be good to use a readable name or if not, at least add a comment somewhere explaining it

Reply:
It's short for `PolyData` but I will change that